packages/teams-js/src/internal/communication.ts

```diff
@@ -52,6 +52,9 @@ class CommunicationPrivate {
   public static promiseCallbacks: {
     [id: number]: Function; // (args[]) => void
   } = {};
+  public static portCallbacks: {
```

Comment:
Why do you need to store portCallbacks (which are still promise based) separately from the normal promiseCallbacks? Is it just to help keep straight that for promiseCallbacks you pass `message.args` only and for portCallbacks you pass only evt.ports[0]?

Reply:
(fwiw I think I like keeping them separate because it reduces the odds of someone passing down ports from the host in ways we aren't expecting and makes sure to keep port passing walled off in its own section of communication, just trying to make sure I'm not missing anything

Reply:
Yes, the separation was to make sure that we can't leak ports unexpectedly in other call types, keeping it very explicit.